README.md

```diff
@@ -84,22 +84,22 @@ Install using `yarn add @energyweb/{package}@preview`
 
 ## Installation
 
-Make sure have latest `yarn` package manager installed.
+Make sure have latest `@microsoft/rush` package manager installed.
```

Comment:
Should we also mention `pnpm` here?

Reply:
Rush automatically installs the preferred pnpm https://github.com/energywebfoundation/origin/blob/5227d9dd8c76cf934fc9a4693d51b2b2c9c09c4a/rush.json#L21-L29